index.html

```diff
@@ -0,0 +1,159 @@
+          <p class="address">
+            @ National Museum of Korea, Art Center Nabi and more
+          </p>
+    </section>
+    <!-- Main Program Section -->
+    <section id="main-program">
+        <p class="title">Main Program</p>
+  <hr />
+  <div class="main-program">
+    <div class="lecture">
+        <img src="./assets/logos/p5.png" alt="lecture logo" />
+        <h3>Lecture</h3>
+        <p>
+          listen to the speakers from various countries about the messeges
+          of sharing and opening.
+        </p>
+    </div>
+    <div class="ccexhibition">
+        <img src="./assets/logos/p1.png" alt="cc Exhibition logo" />
+        <h3>CC Exhibition</h3>
+        <p>
+          Appreciate various creations applying CC license of artists,
+          organized from Art Center Nabi.
+        </p>
+      </div>
+      <div class="forum">
+        <img src="./assets/logos/p2.png" alt="Forum logo" />
+        <h3>Forum</h3>
+        <p>
+          Have the time to share your thoughts and opinions with experts for
+          each topic.
+        </p>
+      </div>
+      <div class="workshop">
+        <img src="./assets/logos/p3.png" alt="workshop logo" />
+        <h3>Workshop</h3>
+        <p>
+          try creating your own work using open source license rather than
+          just watching at it.
+        </p>
+      </div>
+      <div class="ccignite">
+        <img src="./assets/logos/p4.png" alt="cc ignite logo" />
+        <h3>CC Ignite</h3>
+        <p>
+          Get opportunities to network with CC affiliates around the world,
+          also after the summit.
+        </p>
+      </div>
+  </div>
+  <p class="desk">See the whole program</p>
+  <p class="mob">Join CC Global Summit 2022</p>
+    </section>
+    <!-- Featured Speakers Section -->
+    <section id="featured-speaker">
+        <div id="speakers">
+          
+        </div>
+
+    </section>
+    <!-- Partners Section -->
+    <section id="partner">
+        <p class="title">Partners</p>
+        <hr />
+        <div class="partner">
+          <div class="partners">
+            <img src="./assets/logos/amazon.png" alt="amazon logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/deezer.png" alt="deezer logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/linkedin.png" alt="linkedin logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/paypal.png" alt="paypal logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/airbnb.png" alt="airbnb logo" />
+          </div>
+        </div>
+    </section>
+     <!--Footer Section-->
+     <footer id="footer" class="white">
+        <a href="index.html">
+          <img src="./assets/logos/Logo.png" alt="page logo"
+        /></a>
+        <p>2022 Creative Commons Korea. Some Rights Reserved.</p>
+      </footer>
```

Comment:
It looks good on mobile screens, but please follow the exact background color and text design from the original idea.

![image](https://user-images.githubusercontent.com/34779928/196791902-3da9aa32-0634-4c7f-91c3-086a5d3a8fd1.png)